core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -17,20 +17,30 @@
  * collection node domain objects.
  */
 
-// TODO(bhenning): Implement validation functions and related tests.
+// TODO(bhenning): This should be provided by the backend.
```

Comment:
I don't think that's necessarily right for this constant. I prefer it being an Angular constant over continuing to propagate use of GLOBALS (which I thought we eventually wanted to turn-down). I want to create some sort of automated machinery for sending over constants from Python (including the URLs), though I haven't put work toward it yet. This TODO is a partial reminder to complete that work. Leaving it as-is due to the preference of keeping it an Angular constant.